src/app/content/highlights/components/HighlightsPopUp.spec.tsx

```diff
@@ -98,10 +109,11 @@ describe('MyHighlights button and PopUp', () => {
     </Provider>, {createNodeMock});
 
     const isHtmlElement = jest.spyOn(appGuards, 'isHtmlElement');
-
     isHtmlElement.mockReturnValueOnce(true);
 
     act(() => { store.dispatch(openMyHighlights()); });
+    // Force componentDidUpdate()
+    act(() => { store.dispatch(receiveUser(user)); });
```

Comment:
`HighlightsPopup` is no longer class component so probably this store update can be merged with the previous one